docs/sdks/typescript/index.md

```diff
@@ -55,927 +67,740 @@ Each SpacetimeDB client depends on some bindings specific to your module. Create
-| [`{Table}`](#class-table)                       | `{Table}` is a placeholder for each of the generated tables.                 |
-| [`{Reducer}`](#class-reducer)                   | `{Reducer}` is a placeholder for each of the generated reducers.             |
-
-### Class `SpacetimeDBClient`
-
-The database client connection to a SpacetimeDB server.
-
-Defined in [spacetimedb-sdk.spacetimedb](https://github.com/clockworklabs/spacetimedb-typescript-sdk/blob/main/src/spacetimedb.ts):
+```typescript
+module_bindings.DbConnection
+```
 
-| Constructors                                                      | Description                                                              |
-| ----------------------------------------------------------------- | ------------------------------------------------------------------------ |
-| [`SpacetimeDBClient.constructor`](#spacetimedbclient-constructor) | Creates a new `SpacetimeDBClient` database client.                       |
-| Properties                                                        |
-| [`SpacetimeDBClient.identity`](#spacetimedbclient-identity)       | The user's public identity.                                              |
-| [`SpacetimeDBClient.live`](#spacetimedbclient-live)               | Whether the client is connected.                                         |
-| [`SpacetimeDBClient.token`](#spacetimedbclient-token)             | The user's private authentication token.                                 |
-| Methods                                                           |                                                                          |
-| [`SpacetimeDBClient.connect`](#spacetimedbclient-connect)         | Connect to a SpacetimeDB module.                                         |
-| [`SpacetimeDBClient.disconnect`](#spacetimedbclient-disconnect)   | Close the current connection.                                            |
-| [`SpacetimeDBClient.subscribe`](#spacetimedbclient-subscribe)     | Subscribe to a set of queries.                                           |
-| Events                                                            |                                                                          |
-| [`SpacetimeDBClient.onConnect`](#spacetimedbclient-onconnect)     | Register a callback to be invoked upon authentication with the database. |
-| [`SpacetimeDBClient.onError`](#spacetimedbclient-onerror)         | Register a callback to be invoked upon a error.                          |
+A connection to a remote database is represented by the `module_bindings.DbConnection` type. This type is generated per-module, and contains information about the types, tables and reducers defined by your module.
 
-## Constructors
+| Name                                                      | Description                                                                                      |
```

Comment:
This is a table of contents, right? Maybe we should call this "Section" instead of "Name"?

Reply:
Honestly, we could just remove the table header. But I'd prefer to do that later; for now, we're using the `| Name | Description |` formatting in several places, and it's what's listed in the style guide.

Reply:
Okay. I'll just resolve this comment.